pkg/minishift/systemtray/minitray.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package systemtray
+
+import (
+	"fmt"
+	goos "os"
+	"os/exec"
+	"runtime"
+	"syscall"
+
+	"github.com/golang/glog"
+	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/util/os"
+	"github.com/minishift/minishift/pkg/util/os/process"
+)
+
+type MiniTray struct {
```

Comment:
not so sure about the naming ; why `Minitray`... we need a mini one since we server 'petit four' ?

Reply:
Naming was hard, i didn't want to call it systray,.. will change this to systemtray

Reply:
what is the argument NOT to call it system stray?

Reply:
Initially i imported `systray` with alias `systemtray` and wanted to have a different name for the `minishift-system-tray` so named it `minitray`, but will change this to systemtray as now no ambiguity.